PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -78,6 +78,9 @@ class AbstractConnection extends AbstractChannel
     /** @var int */
     protected $heartbeat;
 
+    /** @var float */
+    protected $last_frame;
```

Comment:
That kind of change should come for the whole project and not partially, to be consistent at least.